step.sh

```diff
@@ -168,6 +199,46 @@ function device_farm_run {
     local run_response=$(aws devicefarm schedule-run "${run_params[@]}")
     echo_info "Run started for $run_platform!"
     echo_details "Run response: '${run_response}'"
+
+    # Obtain the ARN for the run from the schedule-run request
+		local run_arn=$(echo "${run_response}" | jq -r .run.arn)
```

Comment:
OK that probably explains why the editor got confused given the mix of whitespace types in the file. I'll correct the lot and submit.